homeassistant/components/iammeter/const.py

```diff
@@ -0,0 +1,219 @@
+    IammeterSensorEntityDescription(
+        key="Voltage_A",
+        name="Voltage_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Current_A",
+        name="Current_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Power_A",
+        name="Power_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ImportEnergy_A",
+        name="ImportEnergy_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ExportGrid_A",
+        name="ExportGrid_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Frequency_A",
+        name="Frequency_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfFrequency.HERTZ,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="PF_A",
+        name="PF_A",
+        icon="mdi:solar-power",
+        native_unit_of_measurement="",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Voltage_B",
+        name="Voltage_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Current_B",
+        name="Current_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="Power_B",
+        name="Power_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfPower.WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ImportEnergy_B",
+        name="ImportEnergy_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
+        key="ExportGrid_B",
+        name="ExportGrid_B",
+        icon="mdi:solar-power",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    IammeterSensorEntityDescription(
```

Comment:
- Device class missing
- We should disable this sensor by default
- missing entity category?
- unit of measurement missing?